src/client/client.cpp

```diff
@@ -1672,6 +1672,10 @@ bool Client::getChatMessage(std::wstring &res)
 	m_chat_queue.pop();
 
 	res = L"";
+	//Save messages in debug.txt
+	std::wstring msg;
+	msg = L"Client: CHAT:"+chatMessage->message;
+	actionstream<<wide_to_utf8(msg)<<std::endl;
```

Comment:
Please do not mix two separate things into one PR, especially feature vs bugfix.
Suggestion: limit this PR to bug fixing.